README.md

```diff
@@ -15,6 +15,86 @@
 A modular set of compiler APIs empowering the next generation of Solidity code analysis and developer tooling.
 Written in Rust and distributed in multiple languages.
 
+Slang analyzes Solidity source code and generates a rich concrete syntax tree (CST) that can be reasoned about. This is a departure from the classic approach of "black-box" compilers, which are handed the input and only their output can be observed.
+
+Slang is not a replacement for solc, the standard Solidity compiler. We do not plan at the moment to support emitting optimized EVM bytecode for use in production. It does not perform formal verification of contracts or Solidity logic in general. However, other tools that serve this purpose are intended to be built on top of it.
+
+[Read the User Guide here.](https://nomicfoundation.github.io/slang/latest/user-guide/01-introduction/)
+
+### Supporting Multiple Solidity Versions
+
+The Solidity programming language has evolved quite a bit since its inception. Some features were introduced, some changed, while some eventually became obsolete and were removed altogether. Developer tooling must be able to understand and consume older contracts that are still being used on the blockchain, written in older versions of Solidity.
+
+Because of that, Slang must be able to reason about different versions of Solidity; how the language grammar, name capture rules, and semantics have changed across different versions. One of our goals is to document differences as part of our [Solidity Grammar](https://nomicfoundation.github.io/slang/latest/solidity-grammar/).
+
+This is why, instead of having to download separate versions of Slang for each Solidity version, you can access the Slang language APIs by simply specifying the Solidity version that you want to work with.
+
+[See our supported versions.](https://nomicfoundation.github.io/slang/latest/solidity-grammar/supported-versions/)
+
+## Installation
+
+You can install the [Slang NPM package](https://www.npmjs.com/package/@nomicfoundation/slang) with the following `npm` command:
+
+```sh
+npm install "@nomicfoundation/slang"
+```
+
+Or if you are using `yarn`:
+
+```sh
+yarn add "@nomicfoundation/slang"
+```
+
+Although Slang is written in Rust, we do not currently support the usage of Slang as a standalone crate.
+
+[Learn more about how to get started with Slang.](https://nomicfoundation.github.io/slang/latest/user-guide/04-getting-started/01-installation/)
```

Comment:
I suggest placing this link (and others like it) in a bullet to indent/for visibility.

```suggestion
- [Learn more about how to get started with Slang.](https://nomicfoundation.github.io/slang/latest/user-guide/04-getting-started/01-installation/)
```

Reply:
I think a few still remain like this still.